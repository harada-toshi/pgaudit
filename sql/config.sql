\! cp sql/config/regress.conf /tmp/config.conf
\! echo "pgaudit.config_file = '/tmp/config.conf'" >> ./tmp_check/data/postgresql.conf
\! rm ./tmp_check/tmp_check.log
\! pg_ctl restart -w -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! sleep 1
\! psql contrib_regression -q -a < sql/config/config.sql
\! rm /tmp/config.conf
\! grep "AUDIT" ./tmp_check/tmp_check.log > results/config.out

