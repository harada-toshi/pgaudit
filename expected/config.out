		format='AUDIT: %command_text'
LOG:  AUDIT: CREATE TABLE test (id int, data text);
LOG:  AUDIT: INSERT INTO test VALUES (1, 'AAA'),(2, 'BBB');
LOG:  AUDIT: UPDATE test SET data = 'XXX' WHERE id = 3;
LOG:  AUDIT: DELETE FROM test WHERE id = 4;
LOG:  AUDIT: SELECT * FROM test;
LOG:  AUDIT: TRUNCATE test;
LOG:  AUDIT: DROP TABLE test;
